inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/core/impl/ThirdPartyClusterServiceImpl.java

```diff
@@ -177,28 +177,59 @@ public Boolean delete(Integer id, String operator) {
     }
 
     @Override
-    public List<DataProxyResponse> getIpList(DataProxyRequest request) {
-        LOGGER.debug("begin to list data proxy by request={}", request);
-        List<ThirdPartyClusterEntity> entityList = thirdPartyClusterMapper.selectByType(Constant.CLUSTER_DATA_PROXY);
-        if (entityList == null || entityList.isEmpty()) {
-            LOGGER.warn("success to list data proxy, but not found anything for request={}", request);
+    public List<DataProxyResponse> getIpList(String clusterName) {
+        LOGGER.debug("begin to list data proxy by clusterName={}", clusterName);
+        ThirdPartyClusterEntity entity = thirdPartyClusterMapper.selectByName(clusterName);
```

Comment:
this change need client must provide clusterName as parameter ?

Reply:
OK, I changed it, if the cluster name was empty, query data proxy cluster by type and select the first one to return.